docs/developing-contracts/state.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_label: State functions
+sidebar_position: 3
+---
+
+# Store data
+
+Persistent data of the Gear contact is stored in the same way as in a classic program and does not require initialization of the external storage.
+
+```rust
+// describe state structure
+pub struct State {
+    votes_received: BTreeMap<String, i32>,
+}
+
+// initialize state itsalfe
+static mut STATE: State = State::new();
```

Comment:
This code is invalid.
1. State has no `new` method explicitly.
2. `new` fn while appear should be `const fn` -> BTreeMap const constructor is unstable and provided by nightly feature